Assignment5/Assignment5.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+var read = function(readFile) {
+  return new Promise((resolve,reject) => {
```

Comment:
@aarushi121 Can we make separate function to write promises only ? `new Promise((resolve,reject) => { }`